Sources/Items/GeneralCollectionViewSectionItem.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  GeneralCollectionViewSectionItem.swift
+//
+//  Copyright © 2017 Rosberry. All rights reserved.
+//
+
+import UIKit.UICollectionView
+
+open class GeneralCollectionViewSectionItem: CollectionViewSectionItem {
```

Comment:
It's better to rename this class to `CollectionViewSectionItem` and add `...Protocol` to the protocol  IMHO. 🤷‍♂️

`General...` 😬

Reply:
I don't think so. I did rename all protocols on purpose and roll it back just because of the simplest implementation of `CollectionViewSectionItem` doesn't make sense for me. Adding word `Protocol` in the end isn't an option, but I agree that the name of `GeneralCollectionViewSectionItem` may be changed.